orquestaconvert/specs/mistral/v2/policies.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019 Extreme Networks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from orquesta.specs.mistral.v2 import base as mistral_v2_spec_base
```

Comment:
another import path problem here i believe
```suggestion
from orquestaconvert.specs.mistral.v2 import base as mistral_v2_spec_base
```

Reply:
Good catch.